.github/MAINTAINER_GUIDE.md

```diff
@@ -0,0 +1,158 @@
+# Maintainer guide
+
+## Keeping track of progress
+
+In order to know what's the progress on the different bugs and features, we use the "Projects" tab.  
+There, several projects are already created to manage the state of issues (To do, In progress, Review in progress, Reviewer approved, Done).
+
+All new issues need to be associated to at least one project.
+
+Planned issues and features are tracked using one of the 2 milestones, allowing to know when a release is possible.  
+Once verified, issues need to be added to a milestone if work needs to be done.
+
+### Projects
+
+| Name | Description |
+|------|-------------|
+| CodeGen | Issues and features associated to Fabulous.CodeGen |
+| Fabulous | Issues and features associated to Fabulous |
+| Fabulous for Xamarin.Forms | Issues and features associated to Fabulous.XamarinForms |
+| Fabulous.StaticView for Xamarin.Forms | Issues and features associated to Fabulous.StaticView.XamarinForms |
+| Fabulous.XF Extensions | Issues and features associated to the extensions of Fabulous.XamarinForms |
+| Repository maintenance | Housekeeping tasks of the repository |
+
+### Milestones
+
+| Name | Description |
+|------|-------------|
+| vCurrent | The current released version supported (non-breaking changes and bug fixes) |
+| vNext | The next version planned to release (new features, breaking changes) |
+
+## Issues
+
+Each new issue needs to be associated at least with 1 type and 1 state labels.  
+Area and platform labels are optionals, and can be used if applicable.
+
+#### General
+
+| Name | Description |
+|------|-------------|
+| duplicate | The issue is a duplicate of another |
+| good first issue | A good issue for newcomers |
+| help wanted | The issue is up-for-grabs |
+| wont fix | The issue won't be fixed |
+
+#### Types
+
+| Name | Description |
+|------|-------------|
+| t/discussion | A subject is being debated |
+| t/enhancement | This is a new feature |
+| t/bug | Something doesn't work as expected |
+| t/housekeeping | This is a repository maintenance task |
+
+#### States
+
+| Name | Description |
+|------|-------------|
+| s/unverified | The issue has not been verified yet |
```

Comment:
Will this label be automatically added for a new issue?
Should we use an automated task for this?